src/forwarders.interfaces.ts

```diff
@@ -1,29 +1,47 @@
-import { SDKEvent, SDKEventCustomFlags } from './sdkRuntimeModels';
+import { SDKEvent, SDKEventCustomFlags, SDKInitConfig } from './sdkRuntimeModels';
 import { Dictionary } from './utils';
-import { IKitConfigs } from './configAPIClient';
-import { IdentityApiData } from '@mparticle/web-sdk';
+import { IFilteringConsentRuleValues, IFilteringEventAttributeValue, IFilteringUserAttributeValue, IKitConfigs, IKitFilterSettings } from './configAPIClient';
+import { Callback, IdentityApiData, Logger, UserIdentities } from '@mparticle/web-sdk';
 import {
     IMParticleUser,
+    ISDKUserAttributes,
     ISDKUserIdentity,
     UserAttributes,
 } from './identity-user-interfaces';
+import { IForwardingStatsData } from './apiClient';
+import { IPixelConfiguration } from './cookieSyncManager';
+import { IdentityAPIMethod } from './identity.interfaces';
+import { AsyncUploader } from './uploaders';
+import { IdentityType } from './types';
 
 // TODO: https://go.mparticle.com/work/SQDSDKS-6035
-export type Kit = Dictionary;
+export interface Kit {
+    // constructor?: new () => IMPForwarder;
+    constructor: () => any;
+    name: string;
+    getId: () => number, // Should this be optional?
```

Comment:
A side loaded kit may not implement `getId` so at a minimum, this should be optional.